### DISCUSSION TITLE
Dart/Flutter getDatabasesPath() throwing exception

### DISCUSSION THREAD

**Discussion by mooseandpebs** [tekartik/sqflite#687] [General]
getDatabasesPath() throwing exception. 
The exception is thrown from the statement:
 final path = await safeInvokeMethod<String?>(methodGetDatabasesPath);

Currently running:
flutter: 2.5.0-6.0.pre
sqfLite: 2.0.0+3

**Comment by alextekartik** [comment]
I was not able to reproduce on latest flutter dev. Does it happen when you try the example application for you? Are you running on iOS or Android (or something else)? Are you in a unit test? Do you have more information on the exception thrown. Are you calling getDatabasesPath after calling `runApp` or `WidgetsFlutterBinding.ensureInitialized();`
